### PR TITLE
Smartphone processor classification

### PR DESCRIPTION
**Pull request by OWAIS-THEGREAT**
# Pull Request for DL-Simplified 💡

## Issue Title : Smartphone Processors Classification

- **Info about the related issue (Aim of the project)** : To classify different processor of smartphones  
- **Name:**Mohammed Owais
- **GitHub ID:** https://github.com/OWAIS-THEGREAT
- **Email ID:** mohammedowais6361@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** SSOC'23

Closes: #223 

### Describe the add-ons or changes you've made 📃

1.First I have imported different libraries like pandas, numpy, tensorflow,keras and etc.
2.Then I analyze the data and filter out the features which has the impact on the output.
3.Then I create two sets one of output and other of input.
4.Then I split data in train and test.
5.Then I apply the three different ANN model to it.
Then take the weighted average to get the best accuracy.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

I have splited the input in two sets test and train.
then predict the output using inbuild functions
then verified with the help of inbuild function(evaluate) to get the accuracy

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Your jupyter notebook's cells are not properly run. Please fix that.
@OWAIS-THEGREAT

**Comment by OWAIS-THEGREAT** [comment]
@abhisheks008 sir, I have double checked that they are running fluently on my system

**Comment by abhisheks008** [comment]
Screenshot 2023-06-10 134257

The cells are not executed.
@OWAIS-THEGREAT
